packages/app/src/elements/login-signup.ts

```diff
@@ -489,6 +489,95 @@ export class LoginOrSignup extends StartForm {
         }
     }
 
+    protected async _migrateLegacyAccount(authResponse: {
+        email: string;
+        legacyData?: PBES2Container;
+        token: string;
+    }): Promise<boolean> {
+        const legacyData = authResponse.legacyData!;
+
+        this._submitEmailButton.start();
+
+        const choice = await alert(
+            $l(
+                "You don't have a Padloc 4 account yet but we've found " +
+                    "an account from an older version. " +
+                    "Would you like to migrate your account to Padloc 4 now?"
+            ),
+            {
+                title: "Account Migration",
+                icon: "user",
+                options: [$l("Migrate"), $l("Learn More"), $l("Cancel")],
+            }
+        );
+
+        if (choice === 1) {
+            window.open("https://padloc.app/help/migrate-v3", "_system");
```

Comment:
Should this be a new guide, or this old one will work?

Reply:
Good question! Since there is no new guide yet, the old one will have to do for now. I'll probably just update the old article when I work on the updated website.